docs/how-to/analytics-dashboard.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Analytics Dashboard
+sidebar_label: Analytics Dashboard
+slug: /analytics-dashboard
+---
+
+# Analytics Dashboard
+
+Amplication's Analytics Dashboard is a powerful tool that provides developers with valuable insights into their application's development process.
+
+The dashboard highlights key metrics related to productivity, efficiency, and code quality to help you understand the impact of using Amplication on your development workflow and the overall return on investment.
+
+To start leveraging the insights provided by the Analytics Dashboard, navigate to the Workspace Overview or Project Overview screen in your Amplication account.
+
+## Productivity Metrics
+
+The Productivity Metrics section provides an overview of the development activity and code generation. This lets developers track their progress and identify trends in their development process using the **Overtime Graph** and **Generated Lines Of Code** sections.
+
+### Overtime Graph
+
+The Overtime Graph is a visual representation of the development activity over a specified period. It displays the number of builds, entities, APIs, and plugins created or updated, allowing developers to track their progress and identify trends in their development process.
+
+Developers can customize the time range to view data from the last month, last 3 months, last year, or a custom date range (not less than 1 month). The default view shows data from the last year, displayed in months.
+
+The Overtime Graph is interactive, allowing developers to select or deselect specific data points (builds, entities, APIs, or plugins) to focus on the metrics that matter most to them.
+
+### Generated Lines of Code (LOC)
+
+The Generated LOC metric provides developers with an overview of the volume of code generated by Amplication. It counts the number of new or updated lines of code (LOC), considering deleted LOC as updated, and calculates the total LOC per build.
+
+This metric helps developers understand the scale of their application and the efficiency of their development process using Amplication.
+
+## Efficiency Metrics
+
+The Efficiency Metrics section highlights the business-related benefits of using Amplication, providing developers with insights into the time saved, cost savings, and code quality improvements achieved through the platform.
+
+### Time Saved
+
+The Time Saved metric estimates the amount of time saved by using Amplication-generated code for your Node.js backend. This metric is based on industry-standard assumptions regarding developer productivity and the efficiency of Node.js backend development.
```

Comment:
That's great. However, in this metric and the next ones as well- I wish to explain our calculations better. What are our assumptions (e.g. "Based on our research and considering the efficiency of Node.js backend developers, we estimate that a developer creates around 8,335 lines of code per month"), and how did we get these numbers (e.g. "We've estimated it to be around 2.8 bugs per 1,000 lines of code (LOC), based on research data, such as findings from Carnegie Mellon University" with a link (?)). 
We don't need to make thorough research here but try to make educated assumptions and explain the logic behind it (hopefully with some evidence).